src/screens/Notifications/notificationsStyles.ts

```diff
@@ -0,0 +1,15 @@
+import EStyleSheet from 'react-native-extended-stylesheet';
+
+export default EStyleSheet.create({
+	container: {
+		flex: 1,
+	},
+	border: {
+		position: 'absolute',
+		top: 0,
+		left: 0,
+		width: 74,
+		height: '100%',
+		backgroundColor: '#f7f7f7',
```

Comment:
to variable, same at  notificationCardStyles.ts,  servicesNotificationCardStyles.ts